xml/deploy_cephadm.xml

```diff
@@ -527,6 +527,23 @@ o- ssh ................................................... [Key Pair set]
   o- private_key ...... [53:b1:eb:65:d2:3a:ff:51:6c:e2:1b:ca:84:8e:0e:83]
   o- public_key ....... [53:b1:eb:65:d2:3a:ff:51:6c:e2:1b:ca:84:8e:0e:83]
 </screen>
+   <note>
+     <para>
+       The resulting exported JSON configuration includes a private SSH key.
+       Take care when dumping this file to ensure that it can not be accessed
+       by others.
+     </para>
+     <para>
+       To ensure this does not occur, you can take an extra precaution by
+       using <command>umask</command>:
+     </para>
+<screen>&prompt.cephuser;umask 077
```

Comment:
this would change mask for all further commands as well.
So we either need to instruct customer to first get current mask with `umask -p` and then restore.
Or execute something like: `(umask 077 && ceph-salt export > cluster.json)` instead.

Reply:
Yes, the latter option would make more sense to me as well. Or how about just using a `chmod 600 cluster.json` right after the file was exported?
```
ceph-salt export > cluster.json && chmod 600 ./cluster.json
```

Reply:
in time between two commands, it is already intercepted by hackers :)

Reply:
Since my suggestion to use `chmod` leaves the file open for prying eyes for a (admittedly very short) period of time, I think Denis' suggestion of using `umask` in a subshell should be preferred:
```
(umask 077 && ceph-salt export > cluster.json)
```

Reply:
not that fast Lenz, I also has assumption that bash interpreter doesn't have any issues and those commands are executed in order )))
So strictly speaking priv key should be exported at all, or by special command and with special perm.

Reply:
Wouldn't we enforce the order by using a semicolon instead?
```
(umask 077 ;  ceph-salt export > cluster.json)
```
I think the question of including the SSH key is out of scope for this PR, let's discuss this in https://github.com/ceph/ceph-salt/issues/288 instead.

Reply:
yes, `&&` also should work, if we assume all underling system works.
But really what I am saying is that we are looking for workarounds instead of fixing the issue,

Reply:
we need bash expert. @smithfarm this thread lacks some comments.

Reply:
I don't think it's a good idea for our documentation to try to teach customers Unix Fundamentals. My opinion is: `umask` is too technical, and we should not mention it here at all.

Reply:
OK, thanks for your suggestion. So in a nutshell, let's stick to `ceph-salt export > cluster.json` and add a note that explains that this is inherently unsafe.

Reply:
+1